string name0
float64 dir0
float64[] vec0

string name1
float64 dir1
float64[] vec1

string name2
float64 dir2
float64[] vec2

string name3
float64 dir3
float64[] vec3

string name4
float64 dir4
float64[] vec4

string name5
float64 dir5
float64[] vec5

